controllers/druid/interface.go

```diff
@@ -0,0 +1,163 @@
+package druid
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/druid-io/druid-operator/apis/druid/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	resourceCreated = "CREATED"
+	resourceUpdated = "UPDATED"
```

Comment:
this can be typed too, just for consistency since the latter const declared are typed